source/query/collation.txt

```diff
@@ -0,0 +1,62 @@
+.. _query-bar-collation:
+
+=================================================
+Set Language Specific Rules for String Comparison
+=================================================
+
+.. default-domain:: mongodb
+
+.. admonition:: Available in |compass| Only
+   :class: important
+
+   .. include:: /includes/fact-unavailable-in-compass-community.rst
+
+Use the :manual:`Collation </reference/collation/>` query bar option
+to specify language-specific rules for string comparison, such as
+rules for lettercase and accent marks.
+
+Set Collation
+-------------
+
+#. Click :guilabel:`Options`.
+
+#. Enter the ``locale`` field in the collation document to specify the
+   `ICU Locale code <http://userguide.icu-project.org/locale>`_ for the
+   desired language in the :guilabel:`Collation` field.
+
+   .. example::
+
+      To use the ``pinyin`` variant of the Chinese collation, use the
+      following collation document:
+
+      .. code-block:: javascript
+
+         { "locale" : "zh@collation=pinyin" }
+
+   As you type, the :guilabel:`Find` button is disabled and the
+   :guilabel:`Collation` label turns red until a valid query is entered.
+
+   .. figure:: /images/querybar/query-collation-error.png
+      :figwidth: 300px
+      :lightbox:
+
+#.
+   .. include:: /includes/run-query.rst
```

Comment:
I think we should move the Note to be under the screenshot. It feels a bit awkward to have the step ("3"), then a semi-related note, then the screenshot associated with the step.